src/adsp_config.c

```diff
@@ -2204,8 +2204,7 @@ static int parse_module(const toml_table_t *toml, struct parse_ctx *pctx,
 		if (strcmp((char *)mod_man->name, "BRNGUP") &&
 		    strcmp((char *)mod_man->name, "BASEFW")) {
 			if (type != i - 1) {
-				log_err(ret, "error: invalid type %d", type);
-				return -EINVAL;
+				log_info(ret, "module type %d is out of order", type);
```

Comment:
I think it's better to say the "module ID type X is not yet supported. Skipping."

Reply:
sure